tao_compiler/mlir/disc/tests/tensorflow_ops/data/sparse_fill_empty_rows_d_f32.mlir

```diff
@@ -0,0 +1,13 @@
+module attributes {tf.versions = {bad_consumers = [], min_consumer = 0 : i32, producer = 0 : i32}} {
+  func.func @main(%indices: tensor<?x?xi64>, %values: tensor<?xf32>, %dense_shape: tensor<?xi64>, %default_value: tensor<f32>) -> (tensor<?x?xi64>, tensor<?xf32>, tensor<?xi1>, tensor<?xi64>) attributes {tf.entry_function = {inputs = "{{INPUTS}}", outputs = "{{OUTPUTS}}", input_placements="{{INPUT_PLACEMENTS}}", output_placements="{{OUTPUT_PLACEMENTS}}"}} {
```

Comment:
better to add partial dynamic shape and static shape test cases as well?

Reply:
Adding a static shape test case will not be possible, since we won't be able to deduce exactly output element numbers  until computation time, thus checks during lowering process will fail

Reply:
add a partial dynamic test case